misc/scripts/remove.sh

```diff
@@ -25,17 +25,29 @@
 # shellcheck disable=SC2034
 { ignore_stack=false; set -o pipefail; trap stacktrace ERR RETURN; }
 
+function clean_rem_pac() {
+    if [[ -e "${METADIR:?}/${_name}" ]]; then
+        sudo rm -f "${METADIR:?}/${_name}"
+    fi
+    if [[ -e "/etc/apt/preferences.d/${_name//./-}-pin" ]]; then
+        sudo rm -f "/etc/apt/preferences.d/${_name//./-}-pin"
+    fi
```

Comment:
```suggestion
    sudo rm -f "${METADIR:?}/${_name}"
    sudo rm -f "/etc/apt/preferences.d/${_name//./-}-pin"
```
No need to check, `rm -f` will succeed anyway